src/test/java/org/opentripplanner/middleware/controllers/api/TrackedTripControllerTest.java

```diff
@@ -178,111 +193,159 @@ void canNotRestartAnOngoingJourney() throws Exception {
                 createPoint(firstStepCoords, 1, NORTH_EAST_BEARING),
                 "IMMEDIATE: Head WEST on Adair Avenue Northeast",
+                TripStatus.BEHIND_SCHEDULE,
                 "Coords near first step should produce relevant instruction"
             ),
             Arguments.of(
+                monitoredTrip,
                 createPoint(firstStepCoords, 4, NORTH_EAST_BEARING),
                 "UPCOMING: Head WEST on Adair Avenue Northeast",
-                "Coords near first step should produce relevant instruction"
+                TripStatus.DEVIATED,
+                "Coords deviated but near first step should produce relevant instruction"
             ),
             Arguments.of(
+                monitoredTrip,
                 createPoint(firstStepCoords, 30, NORTH_EAST_BEARING),
                 "Head to Adair Avenue Northeast",
+                TripStatus.DEVIATED,
                 "Deviated coords near first step should produce instruction to head to first step #1"
             ),
             Arguments.of(
+                monitoredTrip,
                 createPoint(firstStepCoords, 15, NORTH_WEST_BEARING),
                 "Head to Adair Avenue Northeast",
+                TripStatus.DEVIATED,
                 "Deviated coords near first step should produce instruction to head to first step #2"
             ),
             Arguments.of(
+                monitoredTrip,
                 createPoint(firstStepCoords, 20, WEST_BEARING),
                 NO_INSTRUCTION,
+                TripStatus.BEHIND_SCHEDULE,
                 "Coords along a step should produce no instruction"
             ),
             Arguments.of(
+                monitoredTrip,
                 thirdStepCoords,
                 "IMMEDIATE: LEFT on Ponce de Leon Place Northeast",
+                TripStatus.BEHIND_SCHEDULE,
                 "Coords near a not-first step should produce relevant instruction"
             ),
             Arguments.of(
+                monitoredTrip,
                 createPoint(thirdStepCoords, 30, NORTH_WEST_BEARING),
                 "Head to Ponce de Leon Place Northeast",
+                TripStatus.DEVIATED,
                 "Deviated coords near a not-first step should produce instruction to head to step"
+            ),
+            Arguments.of(
+                monitoredTrip,
+                createPoint(destinationCoords, 1, NORTH_WEST_BEARING),
+                "ARRIVED: Monroe Dr NE at Cooledge Ave NE",
+                TripStatus.COMPLETED,
+                "Instructions for destination coordinate"
+            ),
+            Arguments.of(
+                multiLegMonitoredTrip,
+                createPoint(multiItinSecondLegDestCoords, 1, NORTH_WEST_BEARING),
+                "Get off here (Piedmont Ave NE at Monroe Dr)",
+                TripStatus.BEHIND_SCHEDULE,
+                "FIXME: No instruction for bus stop (code assumes we are on the transit leg already)."
```

Comment:
Can this be fixed as part of this update? Is it in scope?

Reply:
@br648  It is not in scope, but I think I mixed up the leg walking to a bus stop and the leg riding the bus. I'll fix that.

Reply:
Addressed in https://github.com/ibi-group/otp-middleware/pull/257/commits/47df409b4cf60612bc28395e483a29570f64d8a8.